src/e2e/pages/landingPage.js

```diff
@@ -72,9 +87,21 @@ export class LandingPage {
     await this.firefoxLogo.click()
   }
 
+  async heroSectionTexts(){
+    return {
+      heroHeaderText: "Find out if your personal information has been compromised",
+      heroDescriptionText: "Stay safe with privacy tools from the makers of Firefox that protect you from hackers and companies that publish and sell your personal information. We'll alert you of any known data breaches, find and remove your exposed info and continually watch for new exposures.",
```

Comment:
Unrelated your PR, but now I'm wondering if the .ftls are using the wrong straight quotes for "We'll". 👀